subprojects/docs/src/main/resources/header.html

```diff
@@ -192,7 +192,7 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
                     <li><a href="../userguide/worker_api.html">Developing Parallel Tasks</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#publishing" aria-expanded="false" aria-controls="developing-plugins">Developing Gradle Plugins</a>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#developing-gradle-plugins" aria-expanded="false" aria-controls="developing-plugins">Developing Gradle Plugins</a>
```

Comment:
🤔 I wonder whether changing these hrefs might break links elsewhere.

Reply:
I think not, since there is already a `href=#publishing` for the: `Publishing Libraries`. It looks like this was copy/pasted and it was not changed.